src/main/java/com/vaadin/demo/component/datepicker/DatePickerFallbackParser.java

```diff
@@ -0,0 +1,36 @@
+package com.vaadin.demo.component.datepicker;
+
+import com.vaadin.demo.DemoExporter; // hidden-source-line
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.data.binder.Result;
+import com.vaadin.flow.router.Route;
+
+import java.time.LocalDate;
+
+@Route("date-picker-fallback-parser")
+public class DatePickerFallbackParser extends VerticalLayout {
+    public DatePickerFallbackParser() {
+        DatePicker datePicker = new DatePicker("Select a date:");
+        // tag::snippet[]
+        datePicker.setFallbackParser((s) -> {
+            LocalDate now = LocalDate.now();
+            if (s.matches("\\+\\d+")) {
+                return Result
```

Comment:
```suggestion
                // Handle input such as "+1"
                return Result
```
Could also remove the `else if` / `else` branches and move the comments above the respective `if` statements where they would be more appropriate.

Reply:
Thanks, applied.